### DISCUSSION TITLE
Multiple inputs for the same value

### DISCUSSION THREAD

**Discussion by n-margalit** [react-hook-form/react-hook-form#7264] [Q&A]
Hi, 
I have a large form and one of the values is presented twice
one time is Checkbox
and the another is radio cutton with the values true or false
I see the value in the form is array ['true', 'on']  or ['false', 'off']
Is there a way to just have this value boolean and presented twice?
TIA

**Comment by JamesWong1999** [comment]
can you share a simple codesandbox?
